t/25-cache.t

```diff
@@ -393,25 +393,25 @@ subtest 'cache directory is corrupted' => sub {
     local $ENV{OPENQA_CACHE_DIR} = $cache_dir->to_string;
     my $db_file = $cache_dir->child('cache.sqlite');
 
-    # New cache dir
+    # New cache dir, force using WAL journaling mode
+    $ENV{OPENQA_CACHE_SERVICE_SQLITE_JOURNAL_MODE} = 'wal';
```

Comment:
It's best to always override env vars with `local` in cases like this. Otherwise it would also affect any new tests after this subtest.